src/style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.imac {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.imac__img {
+  margin: 30px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.imac p {
+  margin: 0;
+  color: #000;
+}
+
+.imac__name {
+  margin: 45px 0 0 14px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.imac__code {
+  margin: 5px 0 0 14px;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.imac__rating {
+  margin: 15px 14px 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.imac__stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars:not(:last-child) {
+  width: 15px;
+  height: 15px;
+  background-image: url(images\star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars:last-child {
+  width: 15px;
+  height: 15px;
+  background-image: url(images\star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.rating {
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.imac__price {
+  margin: 20px 14px 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.price {
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.num {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.imac__buy {
+  margin: 20px 14px 15px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 170px;
+  height: 40px;
+}
+
+.buy {
```

Comment:
at this moment when you hovered on your link - the border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change its position, so it's better if you write border property inside the button.